staff_features/resources/step_definitions/resource_shared.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+Given 'the Resource is opened in edit mode' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+  wait_for_ajax
+end
+
+Given 'the Resource is published' do
+  expect(find('#resource_publish_').checked?).to eq true
+end
+
+Given 'the {string} button is displayed' do |button_text|
+  expect(page).to have_css('a', text: button_text)
+end
+
+Then 'the Resource opens on a new tab in the public interface' do
+  expect(page.windows.size).to eq 2
+  expect(page).to have_text "Resource #{@uuid}"
```

Comment:
Can we be more specific here to test for the public url pattern, something like `#{PUI_URL}/repositories/#{REPO_ID}/resources/#{RESOURCE_ID}`? (this comment duplicates [my comment](https://github.com/archivesspace/e2e-tests/pull/38#discussion_r1892245778) left in PR #38)

Reply:
Yeah right. Thanks for pointing this out Brian. Done.